src/wazuh_modules/vulnerability_scanner/testtool/scanner/fakeAgentData/agentOsData.json

```diff
@@ -15,21 +15,23 @@
         "scan_id": 0,
         "scan_time": "2024/01/06 04:12:44",
         "sysname": "Linux",
+        "triaged": 0,
```

Comment:
Just a note: this column no longer exists.

Reply:
When we have to make the rebase, we can delete it.